src/NuGet.Core/NuGet.PackageManagement/Telemetry/ActionEventBase.cs

```diff
@@ -27,8 +28,8 @@ public class ActionEventBase : TelemetryEvent
                     { nameof(OperationId), operationId },
                     { nameof(PackagesCount), packageCount },
                     { nameof(Status), status },
-                    { nameof(StartTime), startTime.UtcDateTime.ToString("O") },
-                    { nameof(EndTime), endTime.UtcDateTime.ToString("O") },
+                    { nameof(StartTime), startTime.UtcDateTime.ToString("O", CultureInfo.CurrentCulture) },
```

Comment:
In this case, we want a reliable way of comparing datetimes. Then. using a culture-specific datetime could make telemetry data hard to compare. I suggest using an invariant culture.

```suggestion
                    { nameof(StartTime), startTime.UtcDateTime.ToString("O", CultureInfo.InvariantCulture) },
```